### ISSUE TITLE
Screen reader does not announce selected state for GET/POST list items in Swagger API Explorer: A11y_VS Code extension for API Center_Open API Documentation_Name Role Value.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode-azureapicenter#397]
### GitHub Tags
#A11yTCS; #A11ySev2; #Win32; #E2E_VSCodeAPICenter_Win32_March2026; #DesktopApp;#VS Code Extension for API Center; #A11yMAS; #WCAG4.1.2; #Name Role Value;

### Environment Details:
Application Name: VS Code Extension for API Center
Version: 1.109.5 (user setup)

### Repro Steps:

1. Create an API center in azure portal.
2. Open Visual studio Code.
3. Tab till API center extension in the left side panel and press enter.
4. Tab till definition under the Api name which you have created, and press enter.
5. The name which you have provided to the definition will appear for ex-"openapi4", right click on it.
6. Go to "Open API Documentation" using the down arrow and press enter.
7. Swagger petstore screen will appear.
8. Verify that Screen reader does not announce selected state for GET/POST list items in Swagger API Explorer.

### Actual Result:
Screen reader does not announce selected state for GET/POST  list items in Swagger API Explorer.

### Expected Result:
When a checkmark icon for GET/POST is selected or deselected in the Swagger API Explorer, the screen reader should announce the current state (e.g., "GET selected" or "POST not selected")

### User Impact:
Users relying on assistive technologies cannot determine the current state of the list items, leading to confusion about whether GET or POST is selected.

### Attachment:

https://github.com/user-attachments/assets/4b2d297d-808b-46b9-8a54-7361dc811562
